pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/SchemaRegistry.java

```diff
@@ -37,7 +37,7 @@
     CompletableFuture<SchemaVersion> putSchemaIfAbsent(String schemaId, SchemaData schema,
                                                        SchemaCompatibilityStrategy strategy);
 
-    CompletableFuture<SchemaVersion> deleteSchema(String schemaId, String user);
+    CompletableFuture<SchemaVersion> deleteSchema(String schemaId, String user, boolean force);
```

Comment:
Should we remain method `deleteSchema(String schemaId, String user)` then add new method `deleteSchema(String schemaId, String user, boolean force)`?

Reply:
It's an alternative way.  Since this method is not oriented to users(not a breaking change),  IMO, we can modify the original method instead of adding a new one. Adding a new method will leave more duplicated codes, I did see many benefits for it.